web/app/dao/MetricsDAO.java

```diff
@@ -250,6 +250,9 @@ public static JsonNode getMetricListViewMetricNodesByMetricId(Integer metricId)
         return getMetricListViewMetricNodes(metric.dashboardName, metric.group);
     }
 
+    public final static String GET_METRIC_AUTO_COMPLETE_LIST = "SELECT DISTINCT metric_name " +
```

Comment:
@alyiwang can you take a look at this specific commit to see if accurately reflects the current state of the MetricsDAO.java file? This was the only thin that got added in the rebase and that shouldn't be the case.

Thanks.